azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerTestBase.java

```diff
@@ -170,4 +171,8 @@ protected void succeedJobs(final String... jobs) {
     }
   }
 
+  protected void assertEvents(final Type... types) {
```

Comment:
The method name change is fine.

What's the advantage for turning the eventCollector from a local variable into a member variable?
The existing style seems to be more explicit to me since I don't have to navigate to the base class to understand the logic and since this method is protected, I will need to pay attention to see if it is overwritten.